packages/@coorpacademy-player-services/src/videos.js

```diff
@@ -1,11 +1,13 @@
 // @flow strict
 
-import type {DataLayer, VideoProvider} from './definitions';
+import type {DataLayer, VideoProvider, VideoTrack} from './definitions';
 
 type FindUriById = (id: string, provider: VideoProvider) => Promise<string>;
+type FindVideoTracksById = (videoId: string) => Promise<Array<VideoTrack>>;
 
 type VideosService = {|
-  findUriById: FindUriById
+  findUriById: FindUriById,
+  findVideoTracksById: FindVideoTracksById
```

Comment:
As it's video service, `findTracksById` seems better (to be correlant with `findUriById`)